### PR TITLE
Changes readme to follow new structure

### PR DESCRIPTION
**Pull request by ronualdo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,69 @@
...
+[![Build Status](https://travis-ci.org/TwilioDevEd/automated-survey-spark.svg?branch=master)](https://travis-ci.org/TwilioDevEd/automated-survey-spark
 
-Use this example application to create a voice survey using Twilio Voice.
+An application example that implements an Automated Survey using Twilio.
 
-[Read the full tutorial here](https://www.twilio.com/docs/tutorials/walkthrough/automated-survey/java/spark)!
-
-## Quick Start
-### Heroku
+## Herokui
```
Comment (acamino) [inline README.md]: Herokui?

Comment (ronualdo) [inline README.md]: Fixed :P

File: README.md [hunk 2]
```diff
@@ -1,39 +1,69 @@
...
-
-## Quick Start
-### Heroku
+## Heroku
 [![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)  
 Heroku will automatically configure the environment variables necessary to launch this application. Heroku automatically uses a ["Free" dyno](https://www.heroku.com/pricing), and provisions a ["Sandbox" MongoLab](https://mongolab.com/plans/pricing/) instance, so you can play around with this application at no charge.
 
-### Local
-1. Clone this repository.
-2. Run ```mvn install``` to run the accompanying test suite, and build the application.
```
Comment (acamino) [inline README.md]: @ronualdo would you mind replacing ''' by '? _Imagine that ' is a backtick_.

File: README.md (hunk 1 above)
Comment (kathyguerron) [inline README.md]: LGTM

**Pull request merged**